### PR TITLE
Laboratory work #3, Alina Batrasova - 23FPL1

### PR DESCRIPTION
**Pull request by AlinaBatrasova**

### CODE REVIEW

File: lab_3_ann_retriever/tests/advanced_search_engine_test.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # pylint: disable=duplicate-code, protected-access
 """
-Checks the third lab's AdvancedSearchEngine class.
+Checks the third lab's Priority КD Tree Retriever class.
```
Comment (Vasilisa-Blyudova) [inline lab_3_ann_retriever/tests/advanced_search_engine_test.py:3]: why do you have these changes here?

**Review by Vasilisa-Blyudova — COMMENTED** [review]
There's only a little time left, it's time to start

File: lab_3_ann_retriever/tests/advanced_search_engine_test.py (hunk 1 above)
Comment (AlinaBatrasova) [reply lab_3_ann_retriever/tests/advanced_search_engine_test.py:3]: I think it was before some editing of tests because of errors

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Nice implementation of the 3rd lab. But 25 changes!!! have to be merged with main

**Pull request closed without merging**